WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -888,9 +888,11 @@ - (void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withT
                                  completion:nil];
 }
 
+// this method allows this VC to be a valid unwind destination for this selector
 - (IBAction)unwindOutWithSegue:(UIStoryboardSegue *)segue
 {
-    // here so that this VC is a valid unwind destination for this selector
+    // unwind segues don't seem to always clean themselves up 😫
+    [self dismissViewControllerAnimated:NO completion:nil];
```

Comment:
We can call `-dismissViewControllerAnimated` when nothing has been presented without problems. Atm there is only one unwind targetting this selector, so we're good there too.

I'm merging this now, and trying to solve the flicker elsewhere. Need to take a break so I keep some hair 😜

Thanks @frosty